organization/privacy.md

```diff
@@ -33,31 +34,31 @@ For staff and interns, we collect:
 * date of birth
 * contact details: email address, physical adddress and phone number
 * bank details to pay salaries
-* health information to manage sickness absence if applicable
+* other information necessary for proper human resources management
```

Comment:
I don't know if you can do this, since this leaves it completely open. Can we at least give some examples perhaps.